### PR TITLE
feat: update to Volar 2.3 alpha

### PR DESCRIPTION
**Pull request by johnsoncodehk**
Update Volar from 2.1 to 2.3 alpha. 2.3 is mainly API changes and should not cause breaking changes.

### CODE REVIEW

File: yarn.lock
Comment (brandonroberts) [inline yarn.lock:1]: Can we delete this file as we're already using pnpm?

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (johnsoncodehk) [reply yarn.lock:1]: Done!

**Comment by johnsoncodehk** [comment]
@brandonroberts The upgrade is complete!

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Thanks @johnsoncodehk!

**Pull request merged**